axelor-production/src/main/resources/i18n/messages_fr.csv

```diff
@@ -744,6 +745,7 @@
 "The manufacturing order was canceled.","L'ordre de fabrication a été annulé.",,
 "The merge must concern only manufacturing orders with Draft or Planned status, with the same products and same workshop stock location and with all bill of materials compatibles.","Les fusions ne concernent que des produits et des ateliers identiques pour des ordres en statut brouillon ou planifié avec des gammes / nomenclatures compatibles.",,
 "The merge must concern only manufacturing orders with Draft or Planned status, with the same products and with all bill of materials compatibles.","Les fusions ne concernent que des produits identiques pour des ordres en statut brouillon ou planifié avec des gammes / nomenclatures compatibles.",,
+"The operation order is already finished and can not be moved.","L'opération est au statut terminé et ne peux pas être déplacée.",,

```

Comment:
please fix following typos: 
`can not` -> `cannot`
`l'opération [...] ne peux` => `l'opération [...] ne peut`